controllers/rhmi/rhmi_controller.go

```diff
@@ -911,6 +915,13 @@ func (r *RHMIReconciler) preflightChecks(installation *rhmiv1alpha1.RHMI, instal
 		log.Info("STS mode enabled for cluster")
 	}
 
+	if !resources.IsInProw(installation) {
+		err = r.checkClusterPackageAvailablity()
+		if err != nil {
+			log.Infof("error validating cluster package availability", l.Fields{"error": err.Error()})
```

Comment:
we should be returning an error here as well - something like
return result, err
under logging. Otherwise we are just going to continue if the clusterPackageAvailability fails.